packages/mwp-core/src/util/cookieUtils.js

```diff
@@ -11,6 +11,10 @@ export const BROWSER_ID_COOKIE = appConfig.api.isProd
 	? 'MEETUP_BROWSER_ID'
 	: 'MEETUP_BROWSER_ID_DEV';
 
+export const SIFT_SESSION_COOKIE = appConfig.api.isProd
```

Comment:
This is a session ID cookie specific to Sift? Maybe add a comment here as to what this is and how it's being used, why its Sift specific. Link to documentation would be helpful.

Reply:
There has been already discussion on naming convention, for details please ref slack communication: https://meetuphq.slack.com/archives/C099CDFJS/p1549032521177100

Reply:
@padsul I think @jaclynj is requesting a link to docs in a code comment so that future devs understand where this is coming from.